### DISCUSSION TITLE
error: Failed to start player

### DISCUSSION THREAD

**Discussion by 3Sawyer** [streamlink/streamlink-twitch-gui#830] [Q&A] [answered]
Checklist

- [x] I understand the difference between Streamlink and Streamlink Twitch GUI.
- [x] I tried solving my issue by reading the project's wiki first.
- [x] This is not a bug report or feature request and I have read the contribution guides.

Environment details
Operating system and version:
Zorin OS 15.3 based on Ubuntu 18.04 Bionic
64-Bit

Streamlink Twitch GUI version:
streamlink-twitch-gui-v1.13.0-x86_64
Streamlink version:
streamlink-2.4.0-1-cp39-cp39-manylinux2014_x86_64 
Configuration details:

Additional comments:
When i start streamlink from Terminal everything works fine. 

```
 ~$ cd ~/Downloads/Streamlink
 xxx@xxx:~/Downloads/Streamlink$ ./streamlink-2.4.0-1-cp39-cp39-manylinux2014_x86_64.AppImage twitch.tv/example best
 [cli][info] Found matching plugin twitch for URL twitch.tv/example
 [cli][info] Available streams: audio_only, 160p (worst), 360p, 480p, 720p, 720p60, 1080p60 (best)
 [cli][info] Opening stream: 1080p60 (hls)
 [cli][info] Starting player: /snap/bin/vlc
```

In Streamlink Twitch Gui Settings i updated the Streamlink executable to the streamlink appimage file.
But when i try to open a Twitch Stream this Error happens:

> Error: Player misconfiguration
> Please check your player settings.
> Found matching plugin twitch for URL twitch.tv/example
> Available streams: audio_only, 160p (worst), 360p, 480p, 720p60, 1080p60 (best)
> Opening stream: 1080p60 (hls)
> Starting player: /snap/bin/vlc
> error: Failed to start player: /snap/bin/vlc (Process exited prematurely)
> Closing currently open stream...

I would add more Infos but i dont really understand how to create a Logfile from Streamlink Twitch Gui.

**Comment by bastimeyer** [comment, marked as answer]
You're trying to use VLC via snap, which is not supported.
https://github.com/streamlink/streamlink-twitch-gui/issues/785

**Reply by 3Sawyer** [reply]
Is there something i can do? Install VLC a different way? Or use MPV? 
Should i use this command to install mpv? `sudo apt-get install mpv ` Or what can you recommend? I guess when i do this i will download an outdated Version again?

Should i add this PPA before? 
`sudo add-apt-repository ppa:mc3man/mpv-tests`
`sudo apt-get update`

Im so confused what is the right way to install it.

**Reply by bastimeyer** [reply]
> Is there something i can do?

Yes, don't use snap for installing your player. Use regular package management. Good luck
